packages/vue-app/template/components/nuxt-child.js

```diff
@@ -13,7 +13,8 @@ export default {
       default: undefined
     }
   },
-  render (h, { parent, data, props }) {
+  render (_, { parent, data, props }) {
+    const h = parent.$createElement
```

Comment:
Just wondering, what is the benefit of doing this?

Reply:
Making `nuxt-child` invisible and removing a virtual node normally. See https://github.com/vuejs/vue-router/blob/dev/src/components/view.js#L19

Reply:
Interesting (somewhat hidden) Vue feature, thanks for sharing